### ISSUE TITLE
[bug] Failed to open stream: File name too long at...

### ISSUE THREAD

**Issue by mwkcoding** [joelbutcher/socialstream#138]
**Describe the bug**
Seems like the file it saves in the `/tmp` directory is too long for PHP to handle. This is happening when doing Oauth with Google.

Full error (path redacted):
```
file_put_contents(/tmp/AOh14GhYuV61b27SDS4krGM-0npLw506dplelVFFQIuNgKD0IfCnMjIpGgE9BmtHjInKOi9GrJ2Oxv-_KhJR3o3FSUhPUu_sCiJO-pWTggfMphlEszemEa8qh96Xi4E1pacR__MsojaHIVxSPHPOmu0yslmY8gdcIS40CGyAbMj93sY4Hu3ep8h0smc3rlsPbxlE7g6GambFdRP-DvRglquzVO-1vndxtkYGzyGJucBCSeyDB4lFFkA7Hws7XmymFKf-WNI8DDS1sZg1_opwvXVCS4_YYgcfs6_Kyx6EaBLBnytlBfbugpyl75Q9rtXQYsktEX9aAR3vYX9czapSMHx00XB-rai4GpBbi1EgBHh-FT7vscRRKGdhN9hOm57l8u6Q1-4J9iTIeOejD66hRuuXD-4IL2zBUt6P7wyjSVshvah4W794Sh75Szy6YHywBqdBWC-wplvzGg-PHNtj-zfiQZ-6S9ee9oQCoikxI4QUj0zl1PG6TEfyO2yib2pfWP7qHeO1qFktMEW0PtYiMH_byqOGaLZnS0Oh54by8-o8FKIRkzPI9Ic1lKU8jKuLvvT0N6818Gj-aAjSdMoHwQ_bv6xmL_oM_ZFZuDabi60pZw2puYs_IiW64wUoUIuGQ-9Ng92CdzmWqD5ZXmQBGP4dcDIgVrO-BjEgshlG2wybCnM5vxAFqf4wVdCCKzuN7cQAGXn3tf71CP4Aj5TDB5XCK-kWpF9isTzC7jXf6Zr8LIlRF-9BAOMMy1b_z8khRQVcqG0PaQ=s96-c): Failed to open stream: File name too long {"exception":"[object] (ErrorException(code: 0): file_put_contents(/tmp/AOh14GhYuV61b27SDS4krGM-0npLw506dplelVFFQIuNgKD0IfCnMjIpGgE9BmtHjInKOi9GrJ2Oxv-_KhJR3o3FSUhPUu_sCiJO-pWTggfMphlEszemEa8qh96Xi4E1pacR__MsojaHIVxSPHPOmu0yslmY8gdcIS40CGyAbMj93sY4Hu3ep8h0smc3rlsPbxlE7g6GambFdRP-DvRglquzVO-1vndxtkYGzyGJucBCSeyDB4lFFkA7Hws7XmymFKf-WNI8DDS1sZg1_opwvXVCS4_YYgcfs6_Kyx6EaBLBnytlBfbugpyl75Q9rtXQYsktEX9aAR3vYX9czapSMHx00XB-rai4GpBbi1EgBHh-FT7vscRRKGdhN9hOm57l8u6Q1-4J9iTIeOejD66hRuuXD-4IL2zBUt6P7wyjSVshvah4W794Sh75Szy6YHywBqdBWC-wplvzGg-PHNtj-zfiQZ-6S9ee9oQCoikxI4QUj0zl1PG6TEfyO2yib2pfWP7qHeO1qFktMEW0PtYiMH_byqOGaLZnS0Oh54by8-o8FKIRkzPI9Ic1lKU8jKuLvvT0N6818Gj-aAjSdMoHwQ_bv6xmL_oM_ZFZuDabi60pZw2puYs_IiW64wUoUIuGQ-9Ng92CdzmWqD5ZXmQBGP4dcDIgVrO-BjEgshlG2wybCnM5vxAFqf4wVdCCKzuN7cQAGXn3tf71CP4Aj5TDB5XCK-kWpF9isTzC7jXf6Zr8LIlRF-9BAOMMy1b_z8khRQVcqG0PaQ=s96-c): Failed to open stream: File name too long at /Users/[REDACTED]/Code/[REDACTED]/vendor/joelbutcher/socialstream/src/SetsProfilePhotoFromUrl.php:18)
```

**To Reproduce**
1. Fresh install, add Google service and make sure `Features::createAccountOnFirstLogin()` and `Features::providerAvatars()` is enabled.
2. Go to login page
3. Attempt to log in with Google Account with a profile image.

**Expected behavior**
To properly retrieve profile image and set as image on account created.

**Environment context**
- Socialstream version: 3.2.0
- Jetstream stack: Livewire
- Laravel version: 8.59.0
- PHP version: 8.0.9

**Comment by mwkcoding** [comment]
I see that this has been changed on the `3.x` branch but no release has been made. Would be nice with a release

**Comment by joelbutcher** [comment]
Just released a patch in v3.2.1
